profile.html

```diff
@@ -0,0 +1,181 @@
+            />
+          </div>
+          <div class="col">
+            <img
+              src="assets/images/anthony-fomin-oHCBHRuzOyA-unsplash-cropped-downsized.jpg"
+              alt="@usernames profilepicture"
+              class="friendPic rounded-pill img-fluid"
+            />
+          </div>
+          <div class="col">
+            <img
+              src="assets/images/anthony-fomin-oHCBHRuzOyA-unsplash-cropped-downsized.jpg"
+              alt="@usernames profilepicture"
+              class="friendPic rounded-pill img-fluid"
+            />
+          </div>
+          <div class="col">
+            <img
+              src="assets/images/anthony-fomin-oHCBHRuzOyA-unsplash-cropped-downsized.jpg"
+              alt="@usernames profilepicture"
+              class="friendPic rounded-pill img-fluid"
+            />
+          </div>
+        </div>
+        <div class="text-center">
+          <a href="" class="link-dark">View all</a>
+        </div>
+      </section>
+
+      <!-- Posts -->
+      <section class="mx-1 mb-2">
+        <h3>Posts</h3>
+        <div class="card mb-3">
+          <div class="card-body">
+            <div class="d-flex align-items-center mb-3">
+              <img
+                src="assets/images/dmitry-ratushny-T8gLom6k8Jo-unsplash-cropped-downsized.jpg"
+                alt="@usernames profilepicture"
+                class="rounded-pill me-2 postImg"
+              />
+              <h5 class="card-title">@myusername</h5>
+            </div>
+            <h6 class="card-subtitle mb-2 text-muted">31. october 2020</h6>
+            <img
+              src="assets/images/mojor-zhu-Qfdb1GbasNk-unsplash-cropped-resized.jpg"
+              class="card-img-top mb-3"
+              alt="..."
+            />
+            <p class="card-text">
+              Nothing beats fall colors🍁🍂 So excited to start my next project!
+            </p>
+            <i class="bi bi-chat-left-text me-4"></i>
+            <i class="bi bi-suit-heart"></i>
+          </div>
+        </div>
+
+        <div class="card mb-3">
+          <div class="card-body">
+            <div class="d-flex align-items-center mb-3">
+              <img
+                src="assets/images/dmitry-ratushny-T8gLom6k8Jo-unsplash-cropped-downsized.jpg"
+                alt="@usernames profilepicture"
+                class="rounded-pill me-2 postImg"
+              />
+              <h5 class="card-title">@myusername</h5>
+            </div>
+            <h6 class="card-subtitle mb-2 text-muted">15. october 2020</h6>
+            <p class="card-text">Hey! What have you been up to lately? 😀</p>
+            <i class="bi bi-chat-left-text me-4"></i>
+            <i class="bi bi-suit-heart"></i>
+          </div>
+        </div>
+      </section>
+    </main>
+
+    <!-- Post to @usernames wall -->
+    <section
+      class="wallPost d-flex align-items-center fixed-bottom w-100 border-top border-primary"
+      style="max-width: 600px"
```

Comment:
Inline style issue again here, please migrate this to SCSS or use bootstrap classes instead.